tests/ApplicationTest.php

```diff
@@ -3,6 +3,7 @@
 namespace OpenCFP\Test;
 
 use OpenCFP\Application;
+use OpenCFP\Domain\CallForProposal;
```

Comment:
this is unused. did you forget to add something? if not, please remove it to prevent Travis from triggering the Code Sniffer error `no_unused_imports`.